Formula/ceph-client.rb

```diff
@@ -0,0 +1,103 @@
+class CephClient < Formula
+  desc "Rados and RBD CLIs and libraries of the Ceph project"
+  homepage "https://ceph.com"
+  url "git@github.com:ceph/ceph.git", :using => :git, :tag => "v13.2.1", :revision => "5533ecdc0fda920179d7ad84e0aa65a127b20d77"
+  version "mimic-13.2.1"
+
+  depends_on python if MacOS.version <= :snow_leopard
+
+  depends_on :osxfuse
+  depends_on "openssl" => :build
+  depends_on "ccache" => :build
+  depends_on "cmake" => :build
```

Comment:
It already is a build dependency - what are you proposing?

Reply:
i am proposing dropping it as a build dependency.

Reply:
Talked to @tchaikov - turns out he was talking about the ccache dependency on [line 11](https://github.com/Homebrew/homebrew-core/pull/30583/files/d43665ceef95f6cd1e8dc4b6efe1418e812a3989#diff-d37373674c5e3b5815f6ffd5429436e0R11) .

Ccache is only useful if you plan to build the project multiple times in a row - but not useful if you only build it once. 

Since reducing the list of dependencies is a good thing, I'll check tonight if removing it will not break anything and commit that change as well